packages/bitcore-wallet-service/src/lib/pushnotificationsservice.ts

```diff
@@ -37,8 +37,26 @@ const PUSHNOTIFICATIONS_TYPES = {
     filename: 'txp_finally_rejected'
   },
   TxConfirmation: {
-    filename: 'tx_confirmation',
-    notifyCreatorOnly: true
+    filename: 'tx_confirmation'
```

Comment:
notifyCreatorOnly should stay.

Reply:
It is no longer necessary to keep that variable as notifications must also be sent to the creator. That variable was only needed in this part of the code that was removed:
 if (
        (copayer.id == notification.creatorId && notificationType.notifyCreatorOnly) ||
        (copayer.id != notification.creatorId && !notificationType.notifyCreatorOnly)
   )